app/assets/stylesheets/projects.scss

```diff
@@ -0,0 +1,18 @@
+/* button */
+
+.my_button {
+  margin-left: 5px;
+}
+
+#project-members {
+	font-weight: bold;
+	margin: 15px 0 5px 0	
+}
+
+.field {
+	margin-top: 10px;
+}
+
+#financial-details {
+	display: none;
+}
```

Comment:
Please correct all indentation - it's not a bug but we want to keep consistent format.